MODULE.bazel

```diff
@@ -9,4 +9,4 @@ module(
 bazel_dep(name = "bazel_skylib", version = "1.1.1")
 bazel_dep(name = "rules_nodejs", version = "5.5.0")
 bazel_dep(name = "aspect_rules_js", version = "0.13.0")
-bazel_dep(name = "aspect_bazel_lib", version = "1.3.1")
+bazel_dep(name = "aspect_bazel_lib", version = "1.7.0")
```

Comment:
Would we prefer putting the updates in a separate commit? Personally I'd rather not have runtime changes in an examples commit 🤷